src/components/InfoTooltip/InfoTooltip.scss

```diff
@@ -13,7 +14,8 @@
     justify-content: center;
     width: 1rem;
     height: 1rem;
-    background-color: #{$grey};
+    background-color: #{$color-greyscale-8};
+    color: $color-greyscale-12;
```

Comment:
nit: Not a change you've made but could refactor the use of `px` in this file to `rem`

Reply:
Switched `margin-left` to `rem`, I'll check with design about `border-radius`.